### ISSUE TITLE
Email notification sent out when you save an entry type

### ISSUE THREAD

**Issue by johnwbaxter** [barrelstrength/craft-sprout-email#76]
### Description

Email notification sent out when you save an entry type (i appreciate that when you save changes in an entry type, it re-saves every entry)

### Steps to reproduce

Event: When an entry is saved.
Settings: When entry is created, When entry is updated
Tick any section.

Go to settings -> sections -> edit entry type in section

Save entry type.

Imagine if you had 10,000 entries in a section and then you went and modified the entry type! 10,000 email notifications!!!

### Additional info

- Craft version: 3
- PHP version: 7
- Database driver & version: mysql
- Plugins & versions: Sprout email 4.0.0-beta.6

**Comment by pdaleramirez** [comment]
@BenParizek I fixed this. This happens because when saving the section it runs the ResaveElements queue job. 
ResaveElements queue job is an ajax request so i added a condition to not trigger if it is an ajax request and also not trigger it if it is a console request

**Comment by BenParizek** [comment]
@pdaleramirez I've Updated this to use scenarios:

```
if ($event->sender->getScenario() !== Element::SCENARIO_LIVE)
```

**Comment by BenParizek** [comment]
@johnwbaxter This should now be resolved in v4.0.0-beta.9
